index.html

```diff
@@ -5,13 +5,143 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/style.css">
-    <script src="/script.js"></script>
-    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+    <title>World Health Summit</title>
 </head>
 
 <body>
-    <h1>Hello World</h1>
+    <nav class="social">
+        <ul>
+            <li>
+                <a href=""><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+                <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li><a href=""></a></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My Page</a></li>
+            <li><a href="">Logout</a></li>
+        </ul>
+    </nav>
+    <nav class="nav">
+        <div class="logo">
+            <a href="#home"><img src="img/logo.png" alt="World Health Summit logo image"></a>
+        </div>
+        <ul class="nav-menu">
+            <li><a class="nav-link" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#main-program">Program</a></li>
+            <li><a class="nav-link" href="#join">Join</a> </li>
+            <li><a class="nav-link" href="#sponsor">Sponsor</a></li>
+            <li><a class="nav-link" href="#news">News</a></li>
+            <li class="nav-link campaign"><a href="#Campaign">WH Campaign</a></li>
+        </ul>
+    </nav>
```

Comment:
- [ ] kindly make the nav available on all the pages, I failed to navigate back to the main page from the about page.

![no-nav](https://user-images.githubusercontent.com/53124548/176977665-56937b37-61a1-42ee-aac0-52c1d9de4f56.png)

Reply:
@elisha2kyakpo1
Updated the live demo. It was pointing to the previous branch.
Issue fixed.